tests/transactions/tests.py

```diff
@@ -534,14 +534,7 @@ def test_nested_inner_durable(self):
                     pass
 
 
-class DisableDurabiltityCheckTests(TestCase):
-    """
-    TestCase runs all tests in a transaction by default. Code using
-    durable=True would always fail when run from TestCase. This would mean
-    these tests would be forced to use the slower TransactionTestCase even when
-    not testing durability. For this reason, TestCase disables the durability
-    check.
-    """
+class DurableTests(TestCase):
```

Comment:
I noticed that we have the same tests for `DurableTests` and `DurableTransactionTests` so I added `DurableBaseTests`.